docs/guidance/build.md

```diff
@@ -2,6 +2,15 @@
 
 This phase is about creating a new IT system, or updating an existing IT system, to implement your [mapping](map) and publish OCDS data.
 
+Alternatively, if you don't have the capacity to create or update an IT system, you can consider reusing an existing [data collection tool](build/data_collection_tools). If you're reusing an existing tool, this phase is about customizing that tool to meet your needs and working out how to collate and publish your data.
```

Comment:
Can we use a simpler word than collate (used again later)? Not a blocker, just a suggestion.

Reply:
After reading all the content, the only sentence that clarifies this step of collation is: "If you collected your data using the data collection form or spreadsheet, the easiest option is to simply collate the data into a single spreadsheet for publication." Even then, I'm not sure what is involved (I guess it's more obvious after working with the spreadsheets?) Anyway, either omit or expand the content about collation, so that it's more easily understood.

Reply:
I've replaced 'collate' with 'combine' in https://github.com/open-contracting/standard/pull/1061/commits/1ad00bc849bc8c636c7dadeebe7f888a6252a824 and expanded the guidance. There are many approaches to combining spreadsheet data, so the guidance is necessarily generic.